gateway-provider-identity-assertion-common/pom.xml

```diff
@@ -85,8 +85,23 @@
             <groupId>org.eclipse.jetty</groupId>
             <artifactId>jetty-util</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.jboss.shrinkwrap.descriptors</groupId>
```

Comment:
Do we only use these dependencies from the test? If yes, can we change the scope?

Reply:
Good point, I don't know, let me check.

Reply:
Actually, this is compile time, that's why I added scope compile.

Reply:
Isn't compile the default scope in maven? I thought if we don't specify any scope it will be compile. My suggestion was to change it to test scope, if we only use this library from the test code.

Reply:
ahh, let me change it to test scope and see if it runs.

Reply:
Has to be compile time 
```
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  20.275 s
[INFO] Finished at: 2022-05-20T14:30:24-04:00
[INFO] ------------------------------------------------------------------------
[WARNING] The requested profile "install" could not be activated because it does not exist.
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project gateway-provider-identity-assertion-common: Compilation failure
[ERROR] /knox/gateway-provider-identity-assertion-common/src/main/java/org/apache/knox/gateway/identityasserter/common/filter/AbstractIdentityAsserterDeploymentContributor.java:[89,32] cannot access org.jboss.shrinkwrap.descriptor.api.webapp30.WebAppDescriptor
[ERROR]   class file for org.jboss.shrinkwrap.descriptor.api.webapp30.WebAppDescriptor not found
```

Reply:
ah ok, thanks for trying